---
title: Study design & rationale
---

## Study design & rationale

Genome-Wide Association Studies (GWAS) have repeatedly scanned the human genome for associations with thousands of protein levels. The approach has successfully identified numerous causal genetic links to disease by bridging the functional gap in post-GWAS analyses via mendelian randomization, colocalization and other statistical approaches.

::: details Primer on Proteomic GWAS
While GWAS has successfully identified genomic loci associated with numerous conditions like Alzheimer's disease, most identified variants lie in non-coding regions. This makes it difficult to determine the biological mechanism and the specific genes, tissues, or pathways, through which a specific genetic variant affects the disease phenotype.

To resolve this, the field has turned to proteomic GWAS, also known as protein Quantitative Trait Loci (pQTL) analysis. In this approach, instead of linking genetics to a disease, the individual abundances of thousands of proteins are treated as phenotypes of interest. These associations are categorized as either cis-pQTLs, where variants are located near or within the protein's own gene indicating direct regulation, or trans-pQTLs, where distant variants regulate the protein. Mapping cis protein-regulating genetic variants is inherently useful for modern tools like Mendelian Randomization, which can statistically predict whether a drug targeting a specific protein will effectively treat a disease based on genetic evidence.
::: 

However, by considering each multiplexed protein in isolation, the GWAS fails to shed light on the pathways through which these proteins are mediating their effects.